### PR TITLE
Regression test for null image

### PR DESCRIPTION
**Pull request by gcedo**

### CODE REVIEW

File: src/parts/blog-post-image.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ function BlogPostImage({ caption, src, alt }) {
 
 BlogPostImage.propTypes = {
   alt: React.PropTypes.string,
-  caption: React.PropTypes.string.isRequired,
+  caption: React.PropTypes.element,
```
Comment (keithamus) [inline src/parts/blog-post-image.js]: Is this a breaking change? Have we always expected an element? Do you mean to put `React.propTypes.node`?

File: src/parts/blog-post-section.js [hunk 2]
```diff
@@ -10,7 +10,10 @@ function BlogPostSection({ section }) {
 }
 
 BlogPostSection.propTypes = {
-  section: React.PropTypes.string.isRequired,
+  section: React.PropTypes.oneOfType([
+    React.PropTypes.string,
+    React.PropTypes.element,
+  ]).isRequired,
```
Comment (keithamus) [inline src/parts/blog-post-section.js]: Looks like `React.PropTypes.node` would be better here?

File: src/parts/blog-post-image.js (hunk 1 above)
Comment (gcedo) [inline src/parts/blog-post-image.js]: It was required, but the code doesn't comply with this requirement https://github.com/economist-components/component-blog-post/blob/master/src/index.js#L117

In this case we are feeding either `null` or a React Element, so I would be more strict.

File: test/index.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import 'babel-polyfill';
 import BlogPost from '../src';
+import MobileDetect from 'mobile-detect';
```
Comment (keithamus) [inline test/index.js]: If I had known prior to this PR that we were UA sniffing I would have definitely challenged this.

**Pull request merged**